src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -134,6 +134,9 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
 
   /** Boolean value to indicate if the role application should be applied on details list. Set to false by default */
   shouldApplyApplicationRole?: boolean;
+
+  /** Callback for selection zone to indicate whether the item was selected */
+  onItemSelected?: (isSelected: boolean) => void;
```

Comment:
this doesn't make sense to me. What is the use case here? Why would a user want this?